src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/ButtonSamplePage.xaml

```diff
@@ -15,31 +15,187 @@
 				   Foreground="{StaticResource PrimaryBrush}"
 				   Margin="12" />
 
-		<Button Content="CONTAINED"
-				Style="{StaticResource ContainedMaterialButtonStyle}"
-				Margin="12" />
-
-		<Button Content="CONTAINED DISABLED"
-				IsEnabled="False"
-				Style="{StaticResource ContainedMaterialButtonStyle}"
-				Margin="12" />
-
-		<Button Content="OUTLINED"
-				Style="{StaticResource OutlinedMaterialButtonStyle}"
-				Margin="12" />
-
-		<Button Content="OUTLINED DISABLED"
-				IsEnabled="False"
-				Style="{StaticResource OutlinedMaterialButtonStyle}"
-				Margin="12" />
-
-		<Button Content="TEXT"
-				Style="{StaticResource TextMaterialButtonStyle}"
-				Margin="12" />
-
-		<Button Content="TEXT DISABLED"
-				IsEnabled="False"
-				Style="{StaticResource TextMaterialButtonStyle}"
-				Margin="12" />
+		<StackPanel Orientation="Horizontal">
+			<Button Content="CONTAINED"
+					Style="{StaticResource ContainedMaterialButtonStyle}"
+					Margin="12" />
+
+			<Button Style="{StaticResource SeeStyleFlyoutButtonStyle}">
+				<Button.Content>
```

Comment:
I think .Content is the default property for Button so you don't need <Button.Content>. I don't mind if it stays

Reply:
updated